lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -167,6 +167,12 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
 
             boolean dataCorrelates = vcDataCorrelates(userId);
             if (!dataCorrelates && completedF2F(f2fRequest, f2fVc)) {
+                auditService.sendAuditEvent(
+                        new AuditEvent(
+                                AuditEventTypes.IPV_F2F_CORRELATION_FAIL,
+                                configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                                auditEventUser));
+
```

Comment:
Could you also add an info log here something like "F2F return - VC data does not correlate so failing"?